package.json

```diff
@@ -41,7 +41,7 @@
     "postcss": "^5.2.14",
     "postcss-cssnext": "^2.9.0",
     "postcss-import": "^9.1.0",
-    "postcss-url": "^5.1.2",
+    "postcss-url": "^6.3.1",
```

Comment:
Here are the changes: https://github.com/postcss/postcss-url/releases/tag/v6.0.0